### ISSUE TITLE
How to use UTC to get this format -  1605249236712

### ISSUE THREAD

**Issue by TMHDesign** [iamkun/dayjs#1213]
I am attempting to convert from moment.js to dayjs.js 

This is my current variable: today = moment.utc()  --->  1605249236712

When I do this:  var today = dayjs().utc().format();    ---->  2020-11-13T06:50:05Z

I used examples here - [https://day.js.org/docs/en/plugin/utc](url)

**Comment by TMHDesign** [comment]
I believe I figured it out. I used 

dayjs().unix()   I ended up w/ 

1605250909815  moment.js
1605250930       dayjs

Why the different length?

**Comment by TMHDesign** [comment]
Why do I get the same values for unix timeOpen & unix timeClose???

timeOpen = today.format('YYYY')+'-'+today.format('MM')+'-'+today.format('DD')+'T08:00:00-06:00';
timeClose = today.format('YYYY')+'-'+today.format('MM')+'-'+today.format('DD')+'T17:00:00-06:00';

console.log(timeOpen + '\n' + timeClose);

console.log(dayjs().unix(timeOpen))
console.log(dayjs().unix(timeClose))

2020-11-13T08:00:00-06:00
2020-11-13T17:00:00-06:00

1605252149
1605252149

**Comment by iamkun** [comment]
You can try `dayjs.utc().valueOf()`

**Comment by TMHDesign** [comment]
Thank you so much, I really appreciate that. For further reference, where would I have learned about that method .valueOf()  Is that in the documentation?

**Comment by iamkun** [comment]
yes https://day.js.org/docs/en/display/unix-timestamp-milliseconds
